mojaloop/values.yaml

```diff
@@ -14143,6 +14143,8 @@ ml-ttk-cronjob-cleanup:
   testCaseEnvironmentFile: *ttkInputValues
 
 ml-ttk-test-setup:
+  configFileDefaults:
+      labels: "prod-tests"
```

Comment:
I think we shouldn't have it here. By default the GP should run everything on the default mojaloop deployment. We took out the tests we don't need for COMESA usecase in my view. And we can set labels in COMESA deployment.